test/mocks/packages/package-with-contracts.zos.json

```diff
@@ -0,0 +1,8 @@
+{
+  "name": "Herbs",
+  "version": "1.1.0",
+  "contracts": {
+    "Impl": "ImplV1",
+    "AnotherImpl": "AnotherImplV1"
+  }
+}
```

Comment:
can we use `package-with-contracts-version-1` and `package-with-contracts-version-2` ?

Reply:
Renamed package-with-contracts-2 to package-with-contracts-v2, but left this one without changes, since several tests use it without referring to another version.